test/services/activation/services/activationService.test.js

```diff
@@ -2,24 +2,20 @@
 /* eslint-disable max-len */
 const chai = require('chai');
 const chaiAsPromised = require('chai-as-promised');
-const nock = require('nock');
+const sinon = require('sinon');
```

Comment:
What is the difference between sinon and nock?

Reply:
le sinon provides mocks, spies, and stubs, nock only provides http-server-mocking which is not required here